### PR TITLE
Done ex4_js-objects-part2

### PR DESCRIPTION
**Pull request by SkyNicarum**

### CODE REVIEW

File: src/ex4_js-objects-part2/task-01.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function getPropertyValue(propertyName, PcObject) {
```
Comment (SkyNicarum) [inline src/ex4_js-objects-part2/task-01.js]: PcObject not a lowerCamelCase its will be fixed

File: src/ex4_js-objects-part2/task-02.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function getEmptyObj() {
+    let protoobj = Object.create(null)
```
Comment (SkyNicarum) [inline src/ex4_js-objects-part2/task-02.js]: The extra variable is also repaired

Comment (aneelia) [reply src/ex4_js-objects-part2/task-02.js:2]: ????

File: src/ex4_js-objects-part2/task-01.js (hunk 1 above)
Comment (aneelia) [reply src/ex4_js-objects-part2/task-01.js:1]: when?

File: src/ex4_js-objects-part2/task-03.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+function DeleteSpaces(defaultString) {
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-03.js:1]: camelCase

File: src/ex4_js-objects-part2/task-05.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+function inclusionCheck(str , understring){
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-05.js:1]: camelCase. understring - usually `substr`

File: src/ex4_js-objects-part2/task-06.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+function firstLetterToUp(str){
+    return str.replace(/( |^)[а-яёa-z]/g, function(x){ return x.toUpperCase();});
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-06.js:2]: it's very strange. redo

File: src/ex4_js-objects-part2/task-07.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+function stringShorten(string, stringLength) {
+  if (string.length > stringLength) {
+    newstring = string.slice(0, stringLength - 1) + "…";
+    return newstring;
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-07.js:4]: camelCase

File: src/ex4_js-objects-part2/task-08.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+function getLowerCamelCase(string){
+    let newWord = string.split(' ');
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-08.js:2]: why do u use `newWord `? words. result of splitting is array

File: src/ex4_js-objects-part2/task-08.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    for (let i = 0; i < newWord.length; i++) {
+        if (i === 0) {
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-08.js:4]: u dont need it in the loop. u can move it and start loop by i = 1

File: src/ex4_js-objects-part2/task-11.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
+function countSymbols(defaultString) {
+    let symbol, count;
+    let symbolAmount = {};
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-11.js:3]: incorrect

File: src/ex4_js-objects-part2/task-12.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+function RoundingANumber(num1 , num2){
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-12.js:1]: camelCase

File: src/ex4_js-objects-part2/task-12.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
...
+    let result = num1+num2
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-12.js:2]: are you sure about this variable?

File: src/ex4_js-objects-part2/task-08.js (hunk 8 above)
Comment (SkyNicarum) [reply src/ex4_js-objects-part2/task-08.js:4]: image
If start with 1, then it is incorrectly executed and the error in the tests

**Pull request closed without merging**